pkgs/tools/admin/azure-cli/python-packages.nix

```diff
@@ -222,13 +222,8 @@ let
       azure-mgmt-media = overrideAzureMgmtPackage super.azure-mgmt-media "9.0.0" "zip"
         "sha256-TI7l8sSQ2QUgPqiE3Cu/F67Wna+KHbQS3fuIjOb95ZM=";
 
-      azure-mgmt-msi = super.azure-mgmt-msi.overridePythonAttrs (old: rec {
-        version = "6.0.1";
-        src = old.src.override {
-          inherit version;
-          sha256 = "sha256-PPkQmUoBkJ8Su7h9G2/t8dVy/PT3uCYZjlf70fnY2vU=";
-        };
-      });
+      azure-mgmt-msi = overrideAzureMgmtPackage super.azure-mgmt-msi "6.0.1" "zip"
+        "sha256-PPkQmUoBkJ8Su7h9G2/t8dVy/PT3uCYZjlf70fnY2vU=";
```

Comment:
I respect this comment https://github.com/NixOS/nixpkgs/pull/71797#discussion_r1044692955 so it should be ok